wordcloud_style.py

```diff
@@ -17,10 +17,24 @@ def select_colour(self, word, value, word_to_values):
 class QuantizedColourSelector(ColourSelector):
     """Apply colour based on a user provided list."""
 
+    def __init__(self, colour_list, max_value):
```

Comment:
Do we need a max_value parameter?

Note: Currently your code produces an error because QuantizedColourSelector(colour_list) in wordcloud_test.py is missing the max_value parameter.